bundle/terranova/tnresources/app.go

```diff
@@ -0,0 +1,78 @@
+package tnresources
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/cli/libs/structdiff"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/apps"
+)
+
+type ResourceApp struct {
+	client *databricks.WorkspaceClient
+	config apps.App
+}
+
+func NewResourceApp(client *databricks.WorkspaceClient, config resources.App) (*ResourceApp, error) {
+	return &ResourceApp{
+		client: client,
+		config: config.App,
+	}, nil
+}
+
+func (r *ResourceApp) Config() any {
+	return r.config
+}
+
+func (r *ResourceApp) DoCreate(ctx context.Context) (string, error) {
```

Comment:
Why DoCreate and not just Create?

Reply:
Makes it more greppable, distinguish from Create in apply.go